internal/pkg/cli/env_init_test.go

```diff
@@ -52,6 +80,12 @@ func TestInitEnvOpts_Validate(t *testing.T) {
 				initEnvVars: initEnvVars{
 					EnvName:    tc.inEnvName,
 					GlobalOpts: &GlobalOpts{appName: tc.inAppName},
+					EnvProfile: tc.inProfileName,
+					TempCreds: tempCredsVars{
```

Comment:
Can we add a test if this is nil?

Reply:
`TempCreds`'s zero value is the empty `tempCredsVars{}` struct which sets all these credential values to the empty string ""